feat: add new keybinding "b" to run a command in the focused node (#1024)
kyazdani42/nvim-tree.lua: README.md

- Double right click acts like `<C-]>`
- `W` will collapse the whole tree
- `S` will prompt the user to enter a path and then expands the tree to match the path

> - `b` will prompt the user to enter a shel command which is the run inside the focused node (where the cursor is)

Reviewer: Minor:

Looks like we are missing an explanation for `B`

> - `b` will prompt the user to enter a shel command which is the run inside the focused node (where the cursor is)

Reviewer: Minor, typo and a slight reword:

```suggestion
- `b` will prompt the user to enter a vim command which is run on the directory of the focused node (where the cursor is)
```

  { key = "q",                            action = "close" },
  { key = "g?",                           action = "toggle_help" },
  { key = "W",                            action = "collapse_all" },
  { key = "S",                            action = "search_node" },

>   { key = "b",                            action = "run_file_command" }

Reviewer: Minor:

I know it's late and it's another round of review, but personally I would rather have this mapping be under `.` to match vim-fugitive :smile: What do you think?

Reviewer 2: I agree it would make more sense. `b` is not really a good default for `run command`. I'd have set something like `x` or `e` (execute), but `.` would be ok too.

Author: You could achieve that with creating a custom mapping when using nvim-tree :P

`{key = '.', action = 'run_file_command'}`

Reviewer 2: what we mean here is having a default like this would make more sense :smile:

Author: I was just joking ;) I've changed it to `.`
